docs/get-started/getting-started.md

```diff
@@ -373,11 +375,107 @@ operation “count at ...”, you should see the graph of Spark Execs and some o
 
-![Join Example on Spark SQL UI](/docs/img/join-sql-ui-example.png)
+![Join Example on Spark SQL UI](../img/join-sql-ui-example.png)
+
+## Enabling RapidsShuffleManager
+---
+**NOTE**
+
+The _RapidsShuffleManager_ is a beta feature!
+
+---
+
+The _RapidsShuffleManager_ is an implementation of the `ShuffleManager` interface in Apache Spark
+that allows custom mechanisms to exchange shuffle data. The _RapidsShuffleManager_ has two components:
+a spillable cache, and a transport that can utilize _Remote Direct Memory Access (RDMA)_ and high-bandwidth 
+transfers within a node that has multiple GPUs. This is possible because the plugin utilizes 
+[Unified Communication X (UCX)](https://www.openucx.org/) as its transport.
+
+- **Spillable cache**: This store keeps GPU data close by where it was produced in device memory,
+but can spill in the following cases:
+  - _GPU out of memory_: If an allocation in the GPU failed to acquire memory, spill will get triggered
+    moving GPU buffers to host to allow for the original allocation to succeed.
+  - _Host spill store filled_: If the host memory store has reached a maximum threshold 
+    (`spark.rapids.memory.host.spillStorageSize`), host buffers will be spilled to disk until
+    the host spill store shrinks back below said configurable threshold.
+    
+  Tasks local to the producing executor will short-circuit read from the cache.
+
+- **Transport**: Handles block transfers between executors using various means like: _NVLink_, _PCIe_, _Infiniband (IB)_, 
+_RDMA over Converged Ethernet (RoCE)_ or _TCP_, and as configured in UCX, in these scenarios:
+  - _GPU-to-GPU_: Shuffle blocks that were able to fit in GPU memory.
+  - _Host-to-GPU_ and _Disk-to-GPU_: Shuffle blocks that spilled to host (or disk) but will be manifested 
+  in the GPU in the downstream Spark task.
+
+In order to enable the _RapidsShuffleManager_, please follow these steps. If you don't have 
+Mellanox hardware go to *step 2*:
+
+1. If you have Mellanox NICs and an Infiniband(IB) or RoCE network, please ensure you have the 
+[MLNX_OFED driver](https://www.mellanox.com/products/infiniband-drivers/linux/mlnx_ofed), 
+and the [`nv_peer_mem` kernel module](https://www.mellanox.com/products/GPUDirect-RDMA) installed.
+
+    With `nv_peer_mem`, IB/RoCE-based transfers can perform zero-copy transfers directly from GPU memory.
+
+2. Install [UCX 1.8.1](https://github.com/openucx/ucx/releases/tag/v1.8.1). 
+
+
+3. You will need to configure your spark job with extra settings for UCX (we are looking to 
+simplify these settings in the near future). Choose the version of the shuffle manager
+that matches your Spark version. Currently we support
+   - Spark 3.0.0 (com.nvidia.spark.rapids.spark300.RapidsShuffleManager) 
+   - Spark 3.0.1 (com.nvidia.spark.rapids.spark301.RapidsShuffleManager) 
+   - Spark 3.1.0 (com.nvidia.spark.rapids.spark310.RapidsShuffleManager)
```

Comment:
This is talking about shim features which don't exist in 0.1.  We should not update our main doc pages to discuss anything that isn't in the stable release.

Reply:
Pushed a change to remove 0.2 related documentation.